yt_dlp/extractor/nhk.py

```diff
@@ -235,7 +239,7 @@ class NhkVodIE(NhkBaseIE):
         'info_dict': {
             'id': 'qfjay6cg',
             'ext': 'mp4',
-            'title': 'DESIGN TALKS plus - Fishermen’s Finery',
+            'title': 'DESIGN TALKS plus - Fishermen\'s Finery',
```

Comment:
I got
```
Error: 404 Not Found

The page you are looking for may have been moved or expired.
```
in line with the `'skip'` reason, for this test URL.

You may have a misguided linter (note `’` vs `'`).